tests/data/group_system/group_crypto_policies/rule_configure_jre_crypto_policy/commented_out.fail.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_standard
```

Comment:
There is no `standard` profile for JRE. Should there be one?
Because the only profile for JRE is one derived from STIG.

Reply:
I'm okay to leave the rule out of any profile for now. Adding a new profile is totally out of scope for this PR.

Reply:
I have marked this for the `(all)` profile, that was introduced in the recent OpenSCAP.